packages/wonder-blocks-form/components/choice-internal.js

```diff
@@ -74,19 +73,36 @@ export default class ChoiceInternal extends React.Component<Props> {
             return CheckboxCore;
         }
     }
+    handleClick = (e: SyntheticEvent<>) => {
+        const {checked, onChange, variant} = this.props;
+
+        // Radio buttons cannot be unchecked and do not change
+        // if clicked on when checked
+        if (variant === "radio" && checked) {
+            return;
+        }
+        onChange(!checked);
+    };
+    handleLabelClick = (e: SyntheticEvent<>) => {
+        e.preventDefault();
+    };
     render() {
         const {
             label,
             description,
+            // we don't need this to go into coreProps
+            // eslint-disable-next-line no-unused-vars
```

Comment:
Heya, while we're at it let's turn on this option so we don't have to do this anymore!

https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

Reply:
This sounds really good.

Reply:
I'll open a separate PR for this rule since it's really independent of the rest of the changes in this one.

Reply:
Yep, that's fine by me!